packages/edit-site/src/components/sidebar-navigation-screen-details-footer/index.js

```diff
@@ -35,9 +36,9 @@ export default function SidebarNavigationScreenDetailsFooter( {
 	const lastRevisionId =
 		record?._links?.[ 'predecessor-version' ]?.[ 0 ]?.id ?? null;
 	const revisionsCount =
-		record?._links?.[ 'version-history' ]?.[ 0 ]?.count ?? 0;
-	// Enable the revisions link if there is a last revision and there are more than one revisions.
-	if ( lastRevisionId && revisionsCount > 1 ) {
+		recordCount || record?._links?.[ 'version-history' ]?.[ 0 ]?.count;
+	// Enable the revisions link if there is a last revision and there is more than one revision.
```

Comment:
Should this comment be updated? To my understanding, the link is now shown when there is at least 1 revision.

Reply:
The comment is right - it was a mistake on my part to have remove `> 1`. 

I'll reinstate. 

To explain, there's a bit of weird history about why other revision links work this way. The TL;DR is that this link was used for theme assets, e.g., templates, which have no database record until they're edited. 

For these files there's only a "revision" after they're edited twice, which means the revision.php page won't display a proper diff.

Global styles handles the click event via prop so it doesn't need the link.

Reply:
Ref: https://github.com/WordPress/gutenberg/issues/49164

Reply:
Thanks for the detailed explanation.